syne_tune/tuner_callback.py

```diff
@@ -155,7 +155,7 @@ class TensorboardCallback(TunerCallback):
 
     def __init__(
             self,
-            ignore_metrics: Optional[List[str]] = None,
+            ignore_metrics: Optional[List[str]] = [],
```

Comment:
Its important to avoid passing mutable arguments as defaults as it can have side-effects. Can you revert it to None?

```

    def __init__(
            self,
            ignore_metrics: Optional[List[str]] = None
    ):
      self.ignore_metrics = ignore_metrics if ignore_metrics else []
  ```